docs/src/Run/Scalability.md

```diff
@@ -1,3 +1,169 @@
-# Scalability
+# **Scalability Guide for Frequency Gateway**
 
-Coming soon...
+This guide explains how to configure and manage scalability using Kubernetes Horizontal Pod Autoscaler (HPA) for Frequency Gateway, ensuring your services scale dynamically based on resource usage.
+
+---
+
+## **Table of Contents**
+
+- [**Scalability Guide for Frequency Gateway**](#scalability-guide-for-frequency-gateway)
+  - [**Table of Contents**](#table-of-contents)
+  - [**Introduction**](#introduction)
+  - [**Prerequisites**](#prerequisites)
+  - [**Configuring Horizontal Pod Autoscaler**](#configuring-horizontal-pod-autoscaler)
+    - [**Default Autoscaling Settings**](#default-autoscaling-settings)
+    - [**Metrics for Autoscaling**](#metrics-for-autoscaling)
+    - [**Sample Configuration**](#sample-configuration)
+  - [**Resource Limits**](#resource-limits)
+  - [**Verifying and Monitoring Autoscaling**](#verifying-and-monitoring-autoscaling)
+  - [**Troubleshooting**](#troubleshooting)
+    - [HPA Not Scaling Pods](#hpa-not-scaling-pods)
+    - [Scaling Too Slowly or Too Aggressively](#scaling-too-slowly-or-too-aggressively)
+
+---
+
+## **Introduction**
+
+Kubernetes Horizontal Pod Autoscaler (HPA) helps scale your deployment based on real-time resource usage (such as CPU and memory). By configuring HPA for the Frequency Gateway, you ensure your services remain available and responsive under varying loads, scaling out when demand increases and scaling down when resources aren't needed.
+
+---
+
+## **Prerequisites**
+
+Before implementing autoscaling, ensure that:
+
+- Kubernetes metrics server is enabled and running (or another resource metrics provider).
```

Comment:
Replace with "Kubernetes metrics server (or another resource metrics provider) is enabled and running."